plugin.go

```diff
@@ -95,6 +96,7 @@ func (p Plugin) Exec() error {
 	attachment := slack.Attachment{
 		Color:      p.Config.Color,
 		ImageURL:   p.Config.ImageURL,
+		AuthorName: "drone-slack",
```

Comment:
Is it possible to omit this? It shows up as kind of crowded in Slack

Reply:
Where is it appearing for you? I didn't see anything in our slack instance.

Reply:
AFAICT it started today (I guess a latest release got published?). 

![image](https://user-images.githubusercontent.com/3250776/227368378-8dd24728-3f1f-47e3-b943-cd638b9f1553.jpeg)

Reply:
#152